src/utils/func.js

```diff
@@ -0,0 +1 @@
+export const map = f => arr => arr.map(f);
```

Comment:
using such a generic name project-wide might be confusing..

Reply:
but it just functional version of `Array.map`